TESTING.md

```
@@ -0,0 +1,169 @@
+docker compose down test-db
+
+# View logs
+docker compose logs test-db
+
+# Reset database
+pnpm e2e:db:reset
+```
+
+## Test Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)